presto-main/src/main/java/io/prestosql/sql/gen/JoinCompiler.java

```diff
@@ -684,10 +684,10 @@ private void generatePositionNotDistinctFromRowWithPageMethod(
             // At that point, we'll be able to fully deprecate Type.equalTo (and friends) and remove this hack.
             if (type.getJavaType().equals(Slice.class) && (
                     type instanceof CharType ||
-                    type instanceof JsonType ||
-                    type instanceof DecimalType ||
-                    type instanceof VarbinaryType ||
-                    type instanceof VarcharType)) {
+                            type instanceof JsonType ||
+                            type instanceof DecimalType ||
+                            type instanceof VarbinaryType ||
+                            type instanceof VarcharType)) {
```

Comment:
This is less readable than before

Reply:
I agree, but there doesn't seem be a good way to make this work with the formatter. We could make it something like
```java
//noinspection PointlessBooleanExpression
if (type.getJavaType().equals(Slice.class) && (false ||
```
But that seems worse. Do you have a suggestion, or is this simply a comment that you wished the code formatter worked better here?

Reply:
You could also consider changing the formatter setting **Wrapping and Braces > Binary Expressions > Align when multiline**

But this will result in things like
```java
return this.kind == other.kind &&
       Objects.equals(this.value, other.value);
```
```java
if (base.equalsIgnoreCase(StandardTypes.VARCHAR) &&
    (parameters.size() == 1) &&
    parameters.get(0).isLongLiteral() &&
    parameters.get(0).getLongLiteral() == VarcharType.UNBOUNDED_LENGTH) {
```
We'd need to see if Checkstyle would work with that type of formatting rule.

Reply:
I don't have a suggestion, but I was hoping there might be a way to tweak the formatter to do something better. Oh well. Let's leave it as you have it now.